packages/product/integration-tests/utils/services/setup-module-service.ts

```diff
@@ -0,0 +1,113 @@
+import { SqlEntityManager } from "@mikro-orm/postgresql"
```

Comment:
todo: this should not be necessary, everything is already setup in the module.spec.ts